riscv_isac/coverage.py

```diff
@@ -661,14 +713,15 @@ def compute_per_line(instr, cgf, xlen, addr_pairs,  sig_addrs):
     if instr.instr_name in ["fmadd.s","fmsub.s","fnmadd.s","fnmsub.s","fmadd.d","fmsub.d","fnmadd.d","fnmsub.d"]:
         rs3_val = '0x' + (arch_state.f_rf[rs3]).lower()
 
-    if instr.instr_name in ['csrrwi']  or instr.instr_name in ['fsrmi']:
-        arch_state.fcsr = instr.zimm
+    if instr.instr_name in ['csrrwi']:
```

Comment:
Tracking changes to fcsr via the `csrrwi` instruction is not necessary anymore. This can be removed.

Reply:
It is necessary for most of the 3 operand instructions to populate it's respective rm value, as we are recieving the rm from fcsr register.

Reply:
The changes to the fcsr register are updated as a part of the csr updates. The `csrrwi` instruction need not be tracked for this purpose.

Reply:
Having these removed causing an issue while populating the coverpoints. Have it raised as an issue with furthermore inputs, and will fix it seperately.